drivers/led/lp5562.c

```diff
@@ -81,6 +81,11 @@ LOG_MODULE_REGISTER(lp5562);
 #define LP5562_MIN_BRIGHTNESS 0
 #define LP5562_MAX_BRIGHTNESS 100
 
+/* Output current defines in 0.1mA */
+#define LP5562_MIN_CURRENT_SETTING 0
+#define LP5562_DEFAULT_CURRENT_SETTING 175
```

Comment:
default value should be handled in the binding (`default:`) and the description of each property updated accordingly to indicate that the default value is the power-on default value.

Reply:
done